aim2dat/strct/structure_importer.py

```diff
@@ -183,9 +183,11 @@ def import_from_mp(
             of strings (e.g. ``['el_bandstructure', 'el_dos']`` to obtain the electronic band
             structure and the electronic density of states). The default value is ``[]``.
         structure_type : str (optional)
-            Materials project includes the initial and final (relaxed) stucture in the database.
-            The intial or final structure can be queried by setting this attribute
+            Materials project includes the initial and final (relaxed) structure in the database.
+            The initial or final structure can be queried by setting this attribute
             to ``initial`` or ``final``, respectively. The default setting is ``initial``.
+        use_openapi : bool (optional)
+            Whether to use openapi as provider. The default value is ``False``.
```

Comment:
```suggestion
            Whether to use the openapi interface of Materials Project. If set to ``False`` the 
            legacy interface is used. The default value is ``False``.
```

I would slightly rephrase it. The provider is still MP, it is just a different interface.